lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign.tpl.php

```diff
@@ -384,10 +384,12 @@
       </div>
 
       <?php // Organ Donation Modal // ?>
-      <?php if ($register_organ_donor): ?>
-        <div data-modal id="modal-organ-donation" role="dialog">
-            <div class="takeover-container -no-slide"></div>
-        </div>
+      <?php if (module_exists('dosomething_organ_donation')): ?>
```

Comment:
instead of doing a `module_exists` here you should be able to do a check for `isset` on the var below.